opentelemetry/src/metrics/mod.rs

```diff
@@ -179,6 +179,36 @@ pub trait InstrumentProvider {
         )))
     }
 
+    /// creates an instrument for recording independent values.
+    fn u64_gauge(
```

Comment:
this needs to be under the feature flag?

Reply:
This is not user-facing (not `pub`), so can't be left as it is. I have only kept the Meter API to create a Gauge instrument under the feature flag.